app/routes.js

```diff
@@ -24,6 +24,16 @@ module.exports = function(app) {
             .exec(resHandler(res));
     });
 
+    // find unique values for given screen and field
+    app.get('/api/unique/:screen/:field', function(req, res) {
+        Sample
+            .find({'screen': req.params.screen})
+            .distinct(req.params.field, function(err, data) {
```

Comment:
Oh and just so you know why I've done the sorting myself.. Mongo annoyingly doesn't let you sort output in a distinct query. http://mongoosejs.com/docs/api.html#query_Query-sort